build/transpile.js

```diff
@@ -1068,7 +1068,7 @@ class Transpiler {
         const errorImports = []
 
         for (let error in errors) {
-            const regex = new RegExp ("[^\\'\"]" + error + "[^\\'\"]")
```

Comment:
this was a bug, for example, if in ts we had a mention of `InvalidOrderbook` then it ended up importing `InvalidOrder` (matching part) in python. so, we should import errors only with full word match